core_lib/web_helpers/flask/user_auth_middleware.py

```diff
@@ -0,0 +1,15 @@
+from werkzeug.wrappers import Request

```

Comment:
flask also provides us with a `Request` class, check if using that class also gives us the same output and we can save on an additional package dependency

Reply:
https://tedboy.github.io/flask/generated/generated/flask.Request.html